src/index.ts

```diff
@@ -0,0 +1,12 @@
+import {Web3AuthLoginProvider, Web3AuthNetworkType} from './web3auth/types'
+import web3AuthProvider from './web3auth'
+import {
+  web3AuthProviderMetadata,
+} from './constants'
+import {Web3AuthConnection} from './web3auth/connection'
+
+type ConfigArgs = {clientId: string; networkType: Web3AuthNetworkType}
+
+export function config({clientId, networkType}: ConfigArgs) {
```

Comment:
I guess `init` would be amore accurate name for this function, `config` sounds like some static config, but this function seems to be rather initializing the web3auth provider